net/remote/remote-node.sh

```diff
@@ -90,9 +94,25 @@ local|tar)
         echo "${pubkey}: $stakeNodesInGenesisBlock" >> ./solana-node-stakes/fullnode-stakes.yml
       done
     fi
+    rm -rf ./solana-client-stakes
+    mkdir ./solana-client-stakes
+    for i in $(seq 0 $((numBenchTpsClients-1))); do
+      # shellcheck disable=SC2086 # Do not want to quote $benchTpsExtraArgs
+      solana-bench-tps -w ./solana-client-stakes/bench-tps"$i".yml $benchTpsExtraArgs
+      # Skip first line, as it contains header
+      tail -n +2 -q ./solana-client-stakes/bench-tps"$i".yml >> ./solana-client-stakes/client-accounts.yml
+      echo "" >> ./solana-client-stakes/client-accounts.yml
```

Comment:
why this extra newline?

Reply:
Just to make sure that the current output file and next file won't get concatenated on the same line. I had to add it, as it was getting clobbered otherwise.